lib/sle2docker/cli.rb

```diff
@@ -48,6 +48,11 @@ def version
       puts Sle2Docker::VERSION
     end
 
+    def self.ensure_can_access_dockerd
+      output = `docker info 2>&1`
```

Comment:
Can we avoid to shell out? We are already using docker's API to communicate with the daemon. We could use the [Docker#info](http://www.rubydoc.info/github/swipely/docker-api/Docker#info-class_method) for that.

Reply:
Sure it makes sense, it will help us also to simplify the tests and probably make better ones, good point.